src/midi/Makefile.am

```diff
@@ -7,6 +7,8 @@ libmidi_a_SOURCES = \
 	midi_alsa.h \
 	midi_coreaudio.h \
 	midi_coremidi.h \
+	midi_fluidsynth.cpp \
+	midi_fluidsynth.h \
 	midi_handler.h \
```

Comment:
More of a style comment, but we could also knock out the `.cpp` with something like:

```
if USE_FLUIDSYNTH
libmidi_a_SOURCES += midi_fluidsynth.cpp
endif
```

Instead of having the the big `#if C_FLUIDSYNTH` guards in the source, but that works too; and is less autotools work too :-)

Reply:
If we move file selection to the buildsystem, that introduces another manual step in case feature will need to be manually disabled (as was with zlib/libpng recently - I hoped I can knock off one line in `config.h`, but I needed to manually edit the solution files instead :( ).

Also, using ifs for disabling whole file this way is already popular throughout the DOSBox code base (one example e.g. in `src/hardware/ipx.cpp`), so I would prefer to keep existing solution for consistency.